cob_twist_controller/src/inverse_jacobian_calculations/inverse_jacobian_calculation.cpp

```diff
@@ -81,6 +82,24 @@ Eigen::MatrixXd PInvBySVD::calculate(const TwistControllerParams& params,
         // Formula 20 - additional part - numerical filtering for least singular value m
         uint32_t m = singularValues.rows();
         singularValuesInv(m) = singularValues(m) / (pow(singularValues(m), 2) + pow(params.beta, 2) + pow(lambda, 2));
+
+        result = svd.matrixV() * singularValuesInv.asDiagonal() * svd.matrixU().transpose();
+    }
+    else if(params.singular_value_damping)
```

Comment:
can you put those parameters in an enum? Otherwise it is counterintuitive that if `numerical_filtering=true` changing `singular_value_damping` does not change anything